app/views/application_records/show.haml

```diff
@@ -100,6 +100,10 @@ by:
           %em
             If a date is selected, the application will move back onto the
             dashboard at that time.
+        %p#notification.hidden
+          %em
+            Get notified when this happens: 
+          = text_field_tag :email_to_notify, '', placeholder: 'email'
```

Comment:
Placeholders should generally be an example of what will go in the field, so something like `joe@schmo.com` would be good here.